### DISCUSSION TITLE
Connect Cells between different papers

### DISCUSSION THREAD

**Discussion by CuMarioStarlinger** [clientIO/joint#2171] [Q&A]
### Introduction

Is it possible to connect cells between multiple papers/graphs? My idea is to have the main graph in a main paper. But I need a second small paper/graph on the sidebar. In some cases I want to connect an element of the main paper/graph with an element of the sidebar-paper/graph by dragging. 
Is this possible? If yes, how?

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS, JointJS+

**Comment by kumilingus** [comment]
No, you can not connect cells between 2 papers.

- you can have one paper instance, and draw 2 dummy papers inside: https://jsfiddle.net/kumilingus/dscqv1Lu/4/
- you can split the link into 2 parts. each link will be pinned to a point on the border of the corresponding paper. visually it will look like a single link (but you might need to update the link points on zoom and scroll. Also the thickness of the link might need to be updated for different zoom levels).
- draw some line outside of the JointJS (an SVG line drawn on top of both papers, updated regularly).
